packages/azure/changelog.yml

```
@@ -1,3 +1,8 @@
+- version: "1.11.5"
+  changes:
+    - description: Add `event.reason` log field for exact data found in requests that matches a firewall rule.
+      type: enhancement
+      link: ""
```

Link to this PR, used in docs.

```suggestion
      link: https://github.com/elastic/integrations/pull/10007
```